tests/math_tests.cpp

```diff
@@ -613,7 +613,7 @@ TEST_CASE("math and_rm8_r8", "[math]") {
 "jmp _end\n");
     x86Checkpoint check = test.LoadCheckpoint();
     check.SetReg32(EAX, 0x000000A7);
-    check.SetPF(); // odd set 1
+    check.UnsetPF(); // low byte has even number of 1, set 1 
```

Comment:
Any insight into why this wasn't failing before? Maybe a bug in the test suite code?

Reply:
I think the reason is that: the flag registers comparison were not included in the method x86Tester:Compare before. I added those check points with my last commit. https://github.com/qtumproject/x86lib/blob/98f3b6410b32b1fcd5d7b555f373ea03f1f83499/tests/test_helpers.cpp#L106